cmdstanpy/model.py

```diff
@@ -1389,8 +1393,8 @@ def _run_cmdstan(
         logger_prefix = 'CmdStan'
         console_prefix = ''
         if runset.one_process_per_chain:
-            logger_prefix = 'Chain [{}]'.format(idx + 1)
-            console_prefix = 'Chain [{}] '.format(idx + 1)
+            logger_prefix = 'Chain [{}]'.format(idx + runset.chain_ids[0])
+            console_prefix = 'Chain [{}] '.format(idx + runset.chain_ids[0])
```

Comment:
In the one process per chain case there's no reason to assume the chain ids are contiguous in general. Can we just use `runset.chain_ids[idx]`?